src/qubes_fwupdmgr.py

```diff
@@ -776,22 +789,21 @@ def downgrade_firmware(self, usbvm=False, whonix=False):
             downgrade_sha,
             whonix=whonix
         )
-        arch_name = downgrade_url.replace(FWUPD_DOWNLOAD_PREFIX, "")
-        arch_path = os.path.join(FWUPD_DOM0_UPDATES_DIR, arch_name)
         if downgrade_dict[vm_name][device_choice]["Name"] == "System Firmware":
-            path = arch_path.replace(".cab", "")
+            Path(BIOS_UPDATE_FLAG).touch(mode=0o644, exist_ok=True)
+            extarcted_path = self.arch_path.replace(".cab", "")
```

Comment:
@Asiderr I assume typo. But the same one in two places so it works fine

Reply:
Fixed